docs/pages/guides/store-assets.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: Create Store Assets
+description: Learn how to create screenshots and previews for your app's store pages.
+sidebar_title: Store assets
+---
+
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+
+Before you can submit your app to the App Store and Play Store, you need to provide some assets for your store listing page.
+The goal of these images and videos is to give prospective users an idea of what your app experience
+is going to be like.
+
+The most important assets you need to add are the app screenshots. Note that while they are called "app screenshots" on both stores, and they *do* have to include accurate
+visuals of your app, there is no rule citing that these images have to be screenshots taken on specific devices. The stores will have
+requirements on image format and size, but within these restrictions, you can be creative.
+For example, a common approach is to design the assets using a design tool such as Figma and incorporate actual app screenshots
+(or designs) with supplementary messaging.
+
+## Different approaches for creating "screenshots"
+
+There are three commonly used approaches to creating your store screenshots. You can choose whichever approach works best for your app's needs and your resources.
+
+### Option 1: actual screenshots
+
+The most straightforward option: open up your app on a real device and take the screenshots.
+
+**Pros**: straightforward to do; most accurate representation of your app
+
+**Cons**: need to load the app on different devices for a full range of screenshots
+
+
+<ImageSpotlight
+  alt="Screenshots of the Expo Go App Store Listing Page"
+  src="/static/images/guides/expo-go-screenshots.png"
+  style={{ maxWidth: 720 }}
+/>
+
+### Option 2: screenshots within a design
+
+The majority of apps use this approach: it involves taking screenshots of the app (or in some cases using existing
+designs instead of actual screenshots) and embedding them in the store assets along with appropriate messaging.
+
+**Pros:** allows conveying additional messaging within the assets
+
+**Cons:** the assets need to be created using a design program
+
+<ImageSpotlight
+  alt="Screenshots of the Brex App Store Listing Page"
+  src="/static/images/guides/brex-screenshots.png"
+  style={{ maxWidth: 720 }}
+/>
+
+### Option 3: make it fancy
+
+Here you can use elements of your app design to highlight on the store pages, but incorprate them into a design with creative messaging.
+
+**Pros:** you can make your store page really creative and fun
+
+**Cons:** need an experienced designer to create and maintain the assets
+
+<ImageSpotlight
+  alt="Screenshots of theMS Office Store Listing Page"
```

Comment:
```suggestion
  alt="Screenshots of the MS Office Store Listing Page"
```

Reply:
one small concern i have here is that people might think we're implying that office is using expo

Reply:
I don't have a better alternative for now, but I'll keep looking and if I find one, I'll replace the image.